lib/teleterm/api/proto/v1/access_request.proto

```diff
@@ -38,6 +38,7 @@ message AccessRequest {
   repeated string threshold_names = 11;
   // a list of resourceIDs requested in the AccessRequest
   repeated ResourceID resource_ids = 12;
+  repeated Resource resources = 13;
```

Comment:
in the web code I added the `resources` field and then later removed the `resource_ids` field because they are redundant, do you plan to do the same here? I did them in separate PRs to decouple the frontend and backend changes

Reply:
Let's stick a `// TODO(avatus)` in here then.